### PR TITLE
FUNCAKE-84 Export as csv

### PR DESCRIPTION
**Pull request by nomadicoder**
- Add export as csv functionality
- Fixed Solr error when sorting on un-indexed field.

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 class CatalogController < ApplicationController
 
+  before_action(:catalog_params, only: :index)
```
Comment (bibliotechy) [inline app/controllers/catalog_controller.rb]: Is this still needed?

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -253,4 +254,10 @@ class CatalogController < ApplicationController
     # default 'mySuggester', uncomment and provide it below
     # config.autocomplete_suggester = 'mySuggester'
   end
+
+  private
+
+  def catalog_params
```
Comment (bibliotechy) [inline app/controllers/catalog_controller.rb]: Is this still needed?

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (nomadicoder) [reply app/controllers/catalog_controller.rb]: This is needed to permit passing parameters from the Catalog Controller.

File: app/controllers/catalog_controller.rb (hunk 2 above)
Comment (nomadicoder) [reply app/controllers/catalog_controller.rb]: Removed.

**Pull request merged**